doc/glossary.rst

```diff
@@ -0,0 +1,257 @@
+Glossary
+========
+
+.. currentmodule:: nilearn
+
+The Glossary provides short definitions of neuro-imaging concepts as well
+as Nilearn specific vocabulary. 
+
+If you wish to add a missing term, please `create a new issue`_ or
+`open a Pull Request`_.
+
+.. glossary::
+    :sorted:
+
+    
+    ANOVA
+    Anova
+        Analysis of variance.
+
+    AUC
+        Area under the curve.
+
+    BIDS
+    bids
+        `Brain Imaging Data Structure`_ is a simple and easy to adopt way
+        of organizing neuroimaging and behavioral data.
+    
+    BOLD
+    Bold
+    bold
+        Blood oxygenation level dependent.
+
+    CanICA
+        Canonical independant component analysis.
+
+    contrast
+    contrasts
+        A `contrast`_ is a linear combination of variables (parameters or
+        statistics) whose coefficients add up to zero, allowing comparison
+        of different treatments.
+
+    Decoding
+        `Decoding`_ consists in predicting, from brain images, the conditions
+        associated to trial.
+
+    EEG
+        Electroencephalography.
+    
+    EPI
+        Echo-Planar Imaging.
+
+    FDR correction
+        `False discovery rate`_ controlling procedures are designed to control
+        the expected proportion of "discoveries" (rejected null hypotheses)
+        that are false (incorrect rejections of the null).
+
+    FIR
+        Finite impulse response.
```

Comment:
This is a type of free-form temporal filter that is used to link neural activity with hemodynamic response, when there is uncertainty on the true model.